src/api/spec/models/package_spec.rb

```diff
@@ -440,12 +440,22 @@
       before { stub_request(:post, backend_url) }
 
       it { is_expected.to be_truthy }
+
+      it "without error" do
```

Comment:
why? it actually only gave it a name because I had to call `subject` and consequently I had to write it in two lines. :thinking: